doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -451,7 +451,11 @@ zcbor
 Trusted Firmware-M
 ==================
 
-|no_changes_yet_note|
+The minimal TF-M profile no longer silences TF-M logs by default.
+
+.. note::
+   This can be a breaking change if the UART instance used by TF-M is
+   already in use elsewhere, e.g. in modem trace with a UART backend.
```

Comment:
```suggestion
* The minimal TF-M build profile no longer silences TF-M logs by default.

  .. note::
     This can be a breaking change if the UART instance used by TF-M already is in use, for example by modem trace with a UART backend.
```

Reply:
I wasn't able to apply this as it conflicted with Mel's feedback. But I recreated it to the best of my ability.